dftimewolf/lib/processors/turbinia.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+"""Processes cloud artifacts using a remote Turbinia instance."""
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+import os
+import tempfile
+
+from dftimewolf.lib.module import BaseModule
+
+from turbinia import client as turbinia_client
+from turbinia import config
+from turbinia import evidence
+from turbinia import output_manager
+from turbinia import TurbiniaException
+from turbinia.message import TurbiniaRequest
+
+
+class TurbiniaProcessor(BaseModule):
+  """Process cloud disks with remote Turbinia instance.
+
+  Attributes:
+    client: A TurbiniaClient object
+    disk_name: Name of the disk to process
+    instance (string): The name of the Turbinia instance
+    project: The project containing the disk to process
+    region (string): The region Turbinia is in
+    zone: The zone containing the disk to process
```

Comment:
Are `region` and `zone` two different things? If you mean "cloud zone" as in `us-central1-c` for both, then I'd suggest renaming this to `turbinia_zone (string)` and `target_zone (string)`.

Reply:
So, they are two different things.  I renamed them to turbinia_zone and turbinia_region.